src/pymor/reductors/bt.py

```diff
@@ -136,6 +136,8 @@ class BTReductor(GenericBTReductor):
     """
 
     def _gramians(self):
+        if self.fom.sampling_time > 0:
+            raise NotImplementedError
```

Comment:
Why would the standard BT not work?

Reply:
I thought the same but I got an error when testing because `LTIPGReductor.build_rom` in line 340 does not set `sampling_time`. I would like to attend to the BT methods in a dedicated PR once this is merged.

Reply:
Ok, sounds good.